Servers/utils/assessment.utils.ts

```
@@ -19,13 +20,14 @@ export const getAssessmentByIdQuery = async (
 
 export const createNewAssessmentQuery = async (assessment: {
   projectId: number;
-}): Promise<Assessment> => {
+}): Promise<Object> => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)